lib/utils/Database.cpp

```diff
@@ -35,6 +35,7 @@ bool
 Database::create(const std::string& file_name)
 {
     std::cout << "Creating file: " << file_name << std::endl;
+    return true;
```

Comment:
I think this change is already made at #274 . I'm open to @dylan-copeland 's suggestion.